snappy_wrappers/wrappers/bcftools/filter/wrapper.py

```diff
@@ -4,35 +4,13 @@
 
 from snakemake.shell import shell
 
-step = snakemake.config["pipeline_step"]["name"]
-config = snakemake.config["step_config"][step]
-
-if "include" in config or "exclude" in config:
-    include = config.get("include", "")
-    exclude = config.get("exclude", "")
-elif "bcftools" in config and ("include" in config["bcftools"] or "exclude" in config["bcftools"]):
-    include = config["bcftools"].get("include", "")
-    exclude = config["bcftools"].get("exclude", "")
-elif "filter_nb" in snakemake.wildcards.keys():
-    filter_nb = int(snakemake.wildcards["filter_nb"]) - 1
-    include = config["filter_list"][filter_nb]["bcftools"].get("include", "")
-    exclude = config["filter_list"][filter_nb]["bcftools"].get("exclude", "")
-else:
-    include = ""
-    exclude = ""
-if include:
-    include = '--include "' + include + '"'
-if exclude:
-    exclude = '--exclude "' + exclude + '"'
-
-if "filter_name" in config:
-    filter_name = config.get("filter_name", "")
-elif "bcftools" in config and "filter_name" in config["bcftools"]:
-    filter_name = config["bcftools"].get("filter_name", "")
-elif "filter_nb" in snakemake.wildcards.keys():
-    filter_name = "bcftools_{}".format(int(snakemake.wildcards["filter_nb"]))
-else:
-    filter_name = "+"
+params = dict(snakemake.params)["args"]
+filter_name = params["filter_name"]
```

Comment:
I personally prefer having the params available directly, e.g.:
```python
params:
  filter_name=get_filter_name,
  include=get_include_path,
```
but I realize that snappy-pipeline has always supplied them like this instead:
```python
params:
  args=get_args,
```
correct? Probably to avoid param name clashes. Could probably be done with `unpack` or `**`, but no point in changing this now.
(No action needed, just thinking out loud)

Reply:
The pipeline has _often_ supplied them with `args=get_args` or `args=get_params`, as far as I know. I don't know what is the difference between the two calls...